src/material.cc

```diff
@@ -15,14 +15,17 @@ const ResourceType Material::kType = "Material";
 Material::~Material() {}
 
 Material::Ptr Material::Create(Agent* creator, double quantity,
-                               Composition::Ptr c, std::string package_name) {
-  Material::Ptr m(new Material(creator->context(), quantity, c, package_name));
+                               Composition::Ptr c, std::string package_name,
+                               double unit_value) {
+  Material::Ptr m(
+      new Material(creator->context(), quantity, c, package_name, unit_value));
   m->tracker_.Create(creator);
   return m;
 }
 
 Material::Ptr Material::CreateUntracked(double quantity, Composition::Ptr c) {
```

Comment:
Should we allow untracked materials to be given a `unit_value` upon creation?  One can always call `SetUnitValue()` after the fact, but it might be more convenient if we allow it directly during creation?